policyengine_us/parameters/gov/states/co/hcpf/chp/expenses.yaml

```
@@ -0,0 +1,17 @@
+description: Urgent care co-pay Child Health Plan Plus
```

change to match style of other descriptions, and add a label
```suggestion
description: Urgent care co-pay Child Health Plan Plus
```